MapboxNavigationTests/CocoaPodsTest/PodInstall/Podfile

```diff
@@ -0,0 +1,9 @@
+platform :ios, '9.0'
+use_frameworks!
+
+target 'PodInstall' do
+  pod 'MapboxGeocoder.swift', '~> 0.6.0'
+  pod 'MapboxNavigationUI.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :branch => 'BRANCH_NAME'
+  pod 'MapboxNavigation.swift', :git => 'https://github.com/mapbox/MapboxNavigation.swift.git', :branch => 'BRANCH_NAME'
+  pod 'OSRMTextInstructions', :git => 'https://github.com/Project-OSRM/osrm-text-instructions.swift', :branch => 'master'
```

Comment:
Oh this is test file. Still I think all these branches should be moved to commits or releases.